docs/modules/ROOT/pages/how-tos/redis/manual_failover.adoc

```diff
@@ -0,0 +1,23 @@
+= How to Manually Perform a Redis Failover
+
+In certain situations Redis nodes end up in `CrashLoopBackOff` because they receive a wrong (old) leader IP address from the running Sentinel.
+
+In the logs this might look like:
+
+[source,shell]
+----
+sentinel Could not connect to Redis at 10.0.0.20:26379: Connection refused
+----
+
+To resolve this situation, get the IP address of the running leader, connect to a Redis Sentinel Pod, start a `redis-cli` and initiate the failover.
```

Comment:
>get the IP address of the running leader

How?

Sorry, only realized this just now...

Reply:
With the `kubectl get pod -o wide` the IP is in the output. Should I change the sentence somehow to make this clear?